docs/node.md

```diff
@@ -36,5 +36,15 @@ yarn
 yarn build
 ```
 
+It will automatically transpile code and include only necessary fonts for
+target environments specified by [Browserslist config](https://github.com/browserslist/browserslist#environment-variables).
+For example, if you are making a web app for a kiosk with Chrome 68, run
+`BROWSERSLIST="Chrome 68" yarn build` and it will produce build with no
```

Comment:
I ran this command got the following error:
```
error "browserslist#node-releases@^1.0.0-alpha.11" doesn't satisfy found match of "node-releases@1.0.0-alpha.10"
error "browserslist#node-releases" is wrong version: expected "1.0.0-alpha.11", got "1.0.0-alpha.10"
```

Reply:
@kevinbarabash This PR updates some package versions by `yarn.lock`. You need to `yarn install` first. (If you did, maybe due to recent NPM downtime: https://status.npmjs.org/)

Reply:
If there a difference between running `yarn` and `yarn install`?  It looks like the `circleci: test` check is running into this issue too.

Reply:
@kevinbarabash AFAIK, `yarn` and `yarn install` is equivalent: https://yarnpkg.com/en/docs/cli/#toc-default-command. The CircleCI build failures are due to old Yarn version (which #1658 addresses).

Reply:
@kevinbarabash The error seems to be fixed!